### ISSUE TITLE
bottom border radius disappears when popup is open

### ISSUE THREAD

**Issue by kotolapki** [hossein-zare/react-native-dropdown-picker#222]
@hossein-zare borderRadius are individually set per documentation, however, upon opening the popup, the bottom radius disappears (see gif). any ideas? 

```
      style={{
        backgroundColor: theme.colors.ctaMain10,
        borderWidth: 0,
        width: 80,
        borderTopLeftRadius: 10,
        borderTopRightRadius: 10,
        borderBottomLeftRadius: 10,
        borderBottomRightRadius: 10
      }}
```

![bug](https://user-images.githubusercontent.com/74780418/105907508-088bbf80-5ff3-11eb-9f50-fcccbc648f8b.gif)

_Originally posted by @xiaohengpan in https://github.com/hossein-zare/react-native-dropdown-picker/issues/63#issuecomment-767838287_

**Comment by kotolapki** [comment]
@hossein-zare Could you make 'noBottomRadius' styles optional? it will be really useful
<img width="636" alt="Снимок экрана 2021-03-16 в 12 33 30" src="https://user-images.githubusercontent.com/46027917/111266131-4b156400-8654-11eb-9d21-234ca5f36a7e.png">

**Comment by vinay-sayone** [comment]
I'm facing the same issue any solution?

**Comment by hossein-zare** [comment]
Hello, I'll add a new property today to disable the behavior.

**Comment by hossein-zare** [comment]
`noTopRadius` & `noBottomRadius` are added.
The package has to be tested, Please wait until its ready to be released.

**Comment by hossein-zare** [comment]
Hello again,
I released **v4.0.4**, The mentioned props above are available.
```javascript
noTopRadius={false}
noBottomRadius={false}
```

**Comment by Dimasprog** [comment]
Try `disableBorderRadius={false}`
